src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -18,11 +18,11 @@ const char *debug_configuration::prefix = "GPU_Debug: ";
 
 #ifdef GPU_DEBUG_CONFIG
 
-#define GPU_DEBUG_COUT std::cout << cldnn::debug_configuration::prefix
+#define GPU_DEBUG_COUT_ std::cout << cldnn::debug_configuration::prefix
```

Comment:
nit: just asking out of curiosity, why do you change the name of define to GPU_DEBUG_COUT_ ?

Reply:
I newly added GPU_DEBUG_COUT definition in debug_configuraton.hpp, it needs debug_configuration instant (from GPU_DEBUG_LOG_RAW_INT).
But GPU_DEBUG_COUT in cpp was used during instatnt creation, so there's no debug_configuration instant at that time. For the simple, I rename GPU_DEBUG_COUT in cpp.

Reply:
I see. thanks.